renku/ui/cli/log.py

```diff
@@ -208,14 +211,24 @@ def _print_dataset_log(log_entry: DatasetLogViewModel) -> str:
 )
 @click.option("-w", "--workflows", is_flag=True, default=False, help="Show only workflow executions.")
 @click.option("-d", "--datasets", is_flag=True, default=False, help="Show only dataset modifications.")
-def log(columns, format, workflows, datasets):
+@click.option("--no-pager", is_flag=True, help="Don't use pager (less) for output.")
+def log(columns, format, workflows, datasets, no_pager):
     """Show a history of renku workflow and dataset commands."""
     from renku.command.log import log_command
 
+    def _show_text(text: str, pager: bool):
```

Comment:
I believe it's OK to have public names for inner functions since they are visible from the outside anyways.